R/plots.R

```diff
@@ -27,7 +27,11 @@ plotStudy <- function(study, modelID, featureID, plotID, testID = NULL, librarie
     is.null(libraries) || is.character(libraries)
   )
 
-  plots <- getPlots(study, modelID = modelID, libraries = libraries)
+  plots <- list()
+  for (i in seq_along(modelID)) {
+    tempPlots <- (getPlots(study, modelID = modelID[i], libraries = libraries))
```

Comment:
Could you please remove the parentheses surrounding `getPlots()`?

Reply:
Thanks. This is now implemented.